### PR TITLE
Language mode for symbol shape option

### PR DESCRIPTION
**Pull request by graphemecluster**

### CODE REVIEW

File: Cantoboard/Base.lproj/Main.storyboard [hunk 1]
```diff
@@ -277,10 +277,11 @@
                                                     <nil key="textColor"/>
                                                     <nil key="highlightedColor"/>
                                                 </label>
-                                                <segmentedControl opaque="NO" contentMode="scaleToFill" fixedFrame="YES" contentHorizontalAlignment="center" contentVerticalAlignment="top" segmentControlStyle="plain" selectedSegmentIndex="1" translatesAutoresizingMaskIntoConstraints="NO" id="ZM2-oE-LJv">
+                                                <segmentedControl opaque="NO" contentMode="scaleToFill" fixedFrame="YES" contentHorizontalAlignment="center" contentVerticalAlignment="top" segmentControlStyle="plain" selectedSegmentIndex="0" translatesAutoresizingMaskIntoConstraints="NO" id="ZM2-oE-LJv">
                                                     <rect key="frame" x="143" y="6" width="160" height="32"/>
                                                     <autoresizingMask key="autoresizingMask" flexibleMinX="YES"/>
                                                     <segments>
+                                                        <segment title="預設"/>
```
Comment (alex-the-man) [inline Cantoboard/Base.lproj/Main.storyboard:284]: <segment title="根據輸入語言"/>
                                                        <segment title="根據前文"/>
                                                        <segment title="半形"/>
                                                        <segment title="全形"/>

File: CantoboardFramework/Keyboard/InputController.swift [hunk 2]
```diff
@@ -58,7 +58,7 @@ struct KeyboardState: Equatable {
     }
     
     var symbolShape: SymbolShape {
-        symbolShapeOverride ?? (keyboardContextualType == .chinese ? .full : .half)
+        symbolShapeOverride ?? (!keyboardContextualType.isEnglish ? .full : .half)
```
Comment (alex-the-man) [inline CantoboardFramework/Keyboard/InputController.swift:61]: If the keyboard is in mixed mode, the keyboard will always show full width symbols with this change, even if the user has just typed English. Is this the best behavior?

Comment (alex-the-man) [reply CantoboardFramework/Keyboard/InputController.swift:61]: Nevermind, let's stick with this for now

File: Cantoboard/Base.lproj/Main.storyboard (hunk 1 above)
Comment (graphemecluster) [reply Cantoboard/Base.lproj/Main.storyboard]: Modified.

**Pull request merged**